File: app/src/main/java/com/neome/api/meta/base/dto/DefnFieldHyperlinkRow.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.MetaIdField;
import com.neome.api.meta.base.Types.MetaIdSpreadsheet;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class DefnFieldHyperlinkRow extends DefnFieldEditable
{
  @Nullable
  public DefnDtoText displayTextVar;

  @Nullable
  public MetaIdField[] hyperlinkFieldIdSet;

  @Nullable
  public MetaIdSpreadsheet spreadsheetId;
}

File: app/src/main/java/com/neome/api/meta/base/dto/DefnFieldHyperlinkRow.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.MetaIdField
import com.neome.api.meta.base.Types.MetaIdSpreadsheet

interface DefnFieldHyperlinkRow : DefnFieldEditable {
    val displayTextVar: DefnDtoText?
    val hyperlinkFieldIdSet: List<MetaIdField>?
    val spreadsheetId: MetaIdSpreadsheet?
}
